sormas-e2e-tests/src/main/java/org/sormas/e2etests/pojo/web/Contact.java

```diff
@@ -49,4 +49,36 @@
   String relationshipWithCase;
   String descriptionOfHowContactTookPlace;
   String uuid;
+  String classification;
+  String status;
+  String multiDay;
+  LocalDate dateOfFirstContact;
+  String externalId;
+  String externalToken;
+  String reportingDistrict;
+  String identificationSource;
+  String identificationSourceDetails;
+  String category;
+  String prohibitionToWork;
+  String homeBasedQuarantinePossible;
+  String quarantine;
+  String highPriority;
+  String preexistingConditionDiabetes;
+  String preexistingConditionHiv;
+  String preexistingConditionLiver;
+  String preexistingConditionMalignancy;
+  String preexistingConditionChronicPulmonary;
+  String preexistingConditionRenal;
+  String preexistingConditionNeurologic;
+  String preexistingConditionCardiovascular;
+  String additionalRelevantPreexistingConditions;
+  String vaccinationStatusForThisDisease;
```

Comment:
Please remove the preexistingCondition from each variable and let the name from the UI.
An example : `immunodeficiencyIncludingHiv`

Reply:
rezolved imho